localization/mobile-center-distribute/ru-RU/mobile_center_distribute.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="mobile_center_distribute_update_dialog_title">Доступно обновление приложения</string>
+  <string name="mobile_center_distribute_update_dialog_message_optional">Обновление %1$s %2$s (%3$d) доступно для скачивания и установки.</string>
+  <string name="mobile_center_distribute_update_dialog_message_mandatory">Разработчики этого приложения требуют выполнить обновление до %1$s %2$s (%3$d).</string>
```

Comment:
This sounds _much_ worse than https://github.com/Microsoft/mobile-center-sdk-android/blob/develop/sdk/mobile-center-distribute/src/main/res/values-ru/mobile_center_distribute.xml#L5